### PR TITLE
Add colour fields support in Jugaad

### PR DESCRIPTION
**Pull request by nisargjhaveri**
Resolves T98.

### CODE REVIEW

File: src/app/views/file_edit.php [hunk 1]
```diff
@@ -12,7 +12,12 @@
         <link rel="stylesheet" href="<?= base_url() ?>static/styles/common_edit.css">
         <link rel="stylesheet" href="<?= base_url() ?>static/styles/file.css">
         <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css">
-        <script src="https://cdnjs.cloudflare.com/ajax/libs/diff_match_patch/20121119/diff_match_patch.js"></script>
+        <?php //Color picker, used in template_edit.php ?>
+        <!-- Color picker -->
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.4/jquery.min.js"></script>
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/spectrum/1.8.0/spectrum.min.js"></script>
+        <link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/spectrum/1.8.0/spectrum.min.css">
```
Comment (polybuildr) [inline src/app/views/file_edit.php:19]: `type="text/css"`? Isn't that time gone? :P

File: src/app/views/file_edit.php [hunk 2]
```diff
@@ -12,7 +12,12 @@
...
+        <!-- Color picker -->
```
Comment (polybuildr) [inline src/app/views/file_edit.php]: Not "end color picker"?

File: src/app/views/template_edit.php [hunk 3]
```diff
@@ -23,3 +26,23 @@ function get_field($name, $meta, $data) {
         endforeach;
     endif;
 ?>
+<script>
+    /* global $ */
+    $('input[data-type=color]').spectrum({
+        showInput: true,
+        showAlpha: true,
+        showPalette: true,
+        palette: [
```
Comment (polybuildr) [inline src/app/views/template_edit.php:35]: Where is this colour pallette from?

File: src/app/views/file_edit.php (hunk 1 above)
Comment (nisargjhaveri) [inline src/app/views/file_edit.php:19]: Backward compatibility! :P

It is always good to have this, no?

File: src/app/views/template_edit.php (hunk 3 above)
Comment (nisargjhaveri) [inline src/app/views/template_edit.php:35]: https://bgrins.github.io/spectrum/#options-togglePaletteOnly

File: src/app/views/file_edit.php (hunk 2 above)
Comment (nisargjhaveri) [inline src/app/views/file_edit.php]: Will change.

**Comment by nisargjhaveri** [comment]
Updated.

**Comment by polybuildr** [comment]
LGTM. I'd like to test it, though. Will merge after testing.

**Comment by polybuildr** [comment]
I can't seem to log in to Jugaad. Always 403.

**Comment by polybuildr** [comment]
My bad. Didn't create the DB. :P

**Pull request merged**